aiverify-foundation/moonshot
# [MS-256] Modify print statements to logging

## Description

To update all print statements into logging with different logging levels, writing to file, standard logging formats

## Motivation and Context

This is to fulfil the engineering standards which recommend using logging instead of print to provide a standard logging format.

## Type of Change




- chore: Routine tasks, maintenance, or tooling changes








## How to Test

Use the following environment variables to vary the log name, minimum logging level and whether to log to file:
export MS_LOG_NAME=moonshot
export MS_LOG_LEVEL=debug
export MS_LOG_TO_FILE=true

Proceed to checkout its moonshot-data counterpart MS-256
Launch the command line in interactive
Try running run_recipe or run_cookbook or other commands and see the logging happening.

For unit tests:
Run the ./run_unit_test.sh

## Checklist

Please check all the boxes that apply to this pull request using "x":


## Screenshots (if applicable)

[If the changes involve visual modifications, include screenshots or GIFs that demonstrate the changes.]

## Additional Notes

Proceed to checkout its moonshot-data counterpart MS-256

<details>
  <summary>Developer Certificate of Origin</summary>

 ```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
 ```
</details>
